community-new.html

```diff
@@ -600,6 +600,21 @@ <h5 style="margin-bottom: 18px;">Ballerina: A Language to Develop, Consume and C
                 </td>	
                 <td class="cEventURL"><a class="cEventRegistration" href="https://www.meetup.com/sfjava/events/284669431" target="_blank" style="text-align: center;">More info</a></td>	
             </tr>
+            <tr class="event-expiry"  data-expiry="March 24, 2022 19:00:00">	
```

Comment:
What   is this expiry date? Is  it  correct?

Reply:
@ashweni

Reply:
@dinushan20

Reply:
+1

Reply:
Actually  that code was used to automatically unpublished the event after it. At the moment it is not working, For now we can remove this